source/_posts/2017/2017-01-28-why-I-switched-scrutinizer-for-phpstan-and-you-should-too.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: post
+title: "Why I Switched Scrutinizer for PHPStan and You Should Too"
+perex: '''
+    I used Scrutinizer for a few years now for code coverage and code quality. Configuration was far complex, some issues appeared and build kept failing. But I really wanted a code quality checker for my open-source projects and this was the best tool there is.
```

Comment:
Not sure about "this was the best tool there is" construct. What about "this was the best tool available"?